ui/src/pages/ShoeMap.tsx

```diff
@@ -152,6 +148,9 @@ export const ShoeMap: React.FC = () => {
           )}
           <ZoomControl position="bottomright" />
           <AttributionControl position="topright" />
+          <Control position="topleft">
+            <Filter onChange={onTagsChange} />
```

Comment:
IIRC, we put the filter outside the map because we had trouble with the events bubbling through the filter into the map. Can you double check to make sure that this isn't an issue?